airflow_munchkin/client_parser/docstring_parser/google_docstring_parser.py

```diff
@@ -0,0 +1,294 @@
+        for i, _chr in enumerate(line):
+            if i >= indent:
+                return True
+            if not _chr.isspace():
+                return False
+        return False
+
+    def _is_section_header(self) -> bool:
+        section = self._line_iter.peek().lower()
+        match = _SECTION_REGEX.match(section)
+        if match:
+            header_indent = self._get_indent(section)
+            section_indent = self._get_current_indent(peek_ahead=1)
+            return section_indent > header_indent
+        return False
+
+    def _is_section_break(self) -> bool:
+        line = self._line_iter.peek()
+        return (
+            not self._line_iter.has_next()
+            or self._is_section_header()
+            or (
+                self._is_in_section
+                and line
+                and not self._is_indented(line, self._section_indent)
+            )
+        )
+
+    def _parse(self) -> None:
+        self._parsed_sections = []
+        self._consume_empty()
+
+        while self._line_iter.has_next():
+            if self._is_section_header():
+                try:
+                    section = self._consume_section_header()
+                    logging.info('Found section: "%s"', section)
+                    if not section.lower() in self._sections:
+                        logging.info("Skipping section")
+                        self._consume_to_next_section()
+                        continue
+                    logging.info("Parsing section")
+                    self._is_in_section = True
+                    self._section_indent = self._get_current_indent()
+                    sections = self._sections[section.lower()](section)
+                finally:
+                    self._is_in_section = False
+                    self._section_indent = 0
+            else:
+                sections = SectionBrick(
+                    "Text", self._join_sentences(self._consume_to_next_section())
+                )
+            self._parsed_sections.append(sections)
+
+    def _parse_parameters_section(self, section: str) -> SectionBrick:
+        fields = self._consume_fields()
+        return SectionBrick(section, fields)
+
+    def _parse_returns_section(self, section: str) -> SectionBrick:
+        fields = self._consume_returns_section()
+        return SectionBrick(section, fields)
+
+    @staticmethod
+    def _join_sentences(lines: List[str]) -> List[str]:
+        if not lines:
+            return []
+        result = []
+        sentence = None
+        for line in lines:
+            if sentence is None:
+                sentence = line
+            else:
+                if line == "":
+                    result.append(sentence)
+                    sentence = None
+                elif line.startswith("- "):
+                    sentence += f"\n{line}"
+                else:
+                    sentence += " " + line.strip()
+        if sentence is not None and sentence != "":
```

Comment:
I would simplify this:
```suggestion
        if sentence:
```